test/models/user_test.rb

```diff
@@ -1,10 +1,35 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
+  def setup
+    @user = User.new(name: "usr", full_name: "Example User")
+  end
   
   # name validation tests
+  test "name should be present (nonblank)" do
+    @user.name = " " * 6
+    assert_not @user.valid?
+  end
+
+  test "name should have a minimum length" do
```

Comment:
well the description is not very accurate... you are testing if object is invalid with length shorter than 3